x-pack/solutions/observability/plugins/slo/public/context/action_modal.tsx

```diff
@@ -22,12 +23,12 @@ interface BaseAction {
 }
 
 interface SingleAction extends BaseAction {
-  type: 'clone' | 'delete' | 'reset' | 'enable' | 'disable';
+  type: 'clone' | 'delete' | 'reset' | 'enable' | 'disable' | 'purge';
```

Comment:
we don't have a single purge action yet implemented, let's add this when we do implement it